src/components/ActivitiesListItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+const ActivitiesListItem = ({name, day, classNo, teacher, id}) => (
+  <div>
+    <Link to={`/edit/${id}`}>
```

Comment:
I recommend to create constants for routes. In this case we will need to have function returning string.